docs/CODE_GUIDELINES.md

```diff
@@ -622,22 +626,47 @@ class Foo
 };
 ```
 
-### 10.1. Destructors in interfaces
+### 11.5. Destructors in interfaces
 
 A class with any virtual functions should have a destructor that is either public and virtual or else protected and nonvirtual (cf. [ISO C++ guidelines](http://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Rc-dtor-virtual)).
 
-## 10. Other conventions
+### 11.6. Constructor Initialzation Lists
+
+For lines up to length `ColumnLimit` (in `.clang-format`) everything will be on one line, excluding the braces which must be on the following lines.
```

Comment:
Now one has to open another file to get the value of `ColumnLimit`. Why not add the maximum line length to the code guidelines?
From where to where is the number of characters actually counted to compare with `ColumnLimit`?
Will the maximum line length be only used to decide if the everything goes into one line for constructors and ignored everywhere else?

Reply:
Does it matter to spend effort on the coding guidelines that touches formatting?

Nobody should have to care about reading it or thinking about the  formatting.

The columnlimit is currently 100. I find it's a decent line length while still fitting two editors side by side on a 1080p screen for diffs. It does cause issues with trailing comments because clang-format doesn't know how to break a comment and does unnatural formatting to fit the comment.

The columnlimit is from 0-100, it doesn't care about indentation and such and is applied everywhere

Reply:
I could state in Section 1 what the columnlimit is and where it's applied. Then the constructor section can remain clean.

Reply:
Ok, definition of line length at start of formatting section now.